packages/web-components/fast-components/src/select/index.ts

```diff
@@ -1,13 +1,29 @@
 import {
-    Select,
+    Select as FoundationSelect,
     SelectOptions,
     selectTemplate as template,
 } from "@microsoft/fast-foundation";
+import { fillColor, neutralLayerFloating } from "../design-tokens";
 import { selectStyles as styles } from "./select.styles";
 
 /**
- * A function that returns a {@link @microsoft/fast-foundation#Select} registration for configuring the component with a DesignSystem.
- * Implements {@link @microsoft/fast-foundation#selectTemplate}
+ * The FAST select class
+ * @internal
+ */
+export class Select extends FoundationSelect {
+    /**
+     * @internal
+     */
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        fillColor.setValueFor(this, neutralLayerFloating);
```

Comment:
This needs to target the `listbox` not the main control. I recall there was an update to how we did this, but I don't recall the fix. Consult Fluent UI, I think it's current.

Reply:
updated